pkg/shell/shell.scss

```diff
@@ -22,6 +22,7 @@
 @use "page";
 @use "../lib/table.css";
 @use "nav.scss";
+@import "@patternfly/patternfly/components/Select/select.css";
```

Comment:
Does it make a difference to @use the .scss file instead?

Reply:
Yes. CSS doesn't support `@use`; that's a SASS thing. (Note the file extension.)